x-pack/plugins/cloud_security_posture/server/routes/benchmarks/benchmarks.ts

```diff
@@ -201,32 +200,31 @@ export const defineGetBenchmarksRoute = (router: CspRouter, cspContext: CspAppCo
         return response.forbidden();
       }
 
+      const cspContext = await context.csp;
+      const fleetServices = cspContext.fleet;
+
       try {
         const soClient = (await context.core).savedObjects.client;
         const { query } = request;
 
-        const agentService = cspContext.service.agentService;
-        const agentPolicyService = cspContext.service.agentPolicyService;
-        const packagePolicyService = cspContext.service.packagePolicyService;
-
-        if (!agentPolicyService || !agentService || !packagePolicyService) {
-          throw new Error(`Failed to get Fleet services`);
-        }
-
```

Comment:
our types were incorrect and these checks were not really needed as we know those services exists.